odxtools/parameters/tablekeyparameter.py

```diff
@@ -24,6 +24,8 @@ class TableKeyParameter(Parameter):
     table_row_ref: Optional[OdxLinkRef]
 
     def __post_init__(self) -> None:
+        self._table: Optional[Table] = None
+        self._table_row: Optional[TableRow] = None
```

Comment:
I'm bit surprised that this works during normal program execution: the `Table` and `TableRow` classes should only be imported while type checking?
```suggestion
        self._table: "Table"
        self._table_row: Optional["TableRow"] = None
```

Reply:
Since Python 3.9, type evaluation is lazy

Reply:
ah, good to know. That makes me wonder why the CI did not complain (because we should also run all tests on python 3.8...)